### DISCUSSION TITLE
How can I use translatable with simple resources?

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#6200] [Help]
How can I use translatable with [simple resources](https://filamentphp.com/docs/2.x/admin/resources/getting-started#simple-modal-resources)?

**Comment by mstfkhazaal** [comment]
@danharrin 
any update?

**Comment by abdulmannans-joistic** [comment]
Have you found any solution on this?

**Reply by ArtMin96** [reply]
A lot of time has passed, and I don't remember if anything was done.

**Reply by danharrin** [reply]
It is supported now AFAIK.
